src/styles.css

```diff
@@ -69,34 +122,76 @@
   font-size: 40px;
 }
 
-.orange-button {
-  background-color: var(--orangeButton);
+.blue-button,
+.orange-button,
+.result-box {
+  background-color: transparent;
+  color: white;
+  border: 1px solid var(--reactBlue);
 }
 
-.gray-button {
-  background-color: var(--grayButton);
+.orange-button:hover,
+.orange-button:active {
+  background-color: var(--orangeButton);
 }
 
-.orange-button:hover,
-.gray-button:hover {
-  filter: brightness(85%);
+.blue-button:hover,
+.blue-button:active {
+  background-color: var(--reactBlue);
 }
 
 .large {
   grid-column: span 2;
 }
 
+/* <Quote /> */
+.quote-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 80vh;
+  font-size: 30px;
+}
+
+.quote {
+  text-align: center;
+  width: 500px;
+}
+
+.quote-section button {
+  background-color: var(--reactBlue);
+  border-style: none;
+  padding: 10px 50px;
+  font-weight: bolder;
+  border-radius: 5px;
+}
+
+.quote-section button:hover,
+.quote-section button:active {
+  background-color: var(--orangeButton);
+}
+
```

Comment:
- [ ] Good work with your design :+1:  But please make sure that your website layout matches the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  

#### Given wireframe
![Screenshot from 2022-11-01 21-19-20](https://user-images.githubusercontent.com/6615394/199344130-e63e9be9-9ede-49ae-a7b8-5ba845fff627.jpg)

#### Your design
![Screenshot from 2022-11-01 21-15-38](https://user-images.githubusercontent.com/6615394/199344134-a9c4ac59-5a97-4f5e-b6f8-052d0eac1f76.jpg)